### PR TITLE
Fix a lot of bugs ; more translateable ; print widget ; use redirect for single files

### PR DESCRIPTION
**Pull request by gsantner**
I extracted the fixes/changes I made that came up when using it in my jekyll project. It is in use since June like this and works very fine. I think it is good if the project also benefits from the fixes.

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -1,20 +1,27 @@
...
 markdown: kramdown
 kramdown:
     input: GFM
 
+#### Colors
+colors:
+  primary: "#2ca6cb"
+  primary_dark: "#2ca6cb"
+  accent: "#b8b8b8"
+
```
Comment (gsantner) [inline _config.yml]: these are the exact colors used by jekyll-jacman in the current verison

**Comment by gsantner** [comment]
@Simpleyyt any remarks?

**Comment by Simpleyyt** [comment]
It too many changes.

**Comment by gsantner** [comment]
And what is not what you want fixed/updated/changed?


It seems quite big, but it isnt that much. the big one is the css file which is 99% just a readable reformat, and readd. The file is actually missing on the master branch, seems to be accidently removed by some merge. main.css is included in header html, and not checked in your master branch. (2700/2900)

**Comment by gsantner** [comment]
Currently still using this, quite happy with the patches above. Without the changes a lot of static stuff / paths do not work correctly.

**Comment by gsantner** [comment]
Sad to see that the much work done here isn't getting any attention.
I'm maintaining a fork of this and use my theme in multiple pages.

**Comment by Simpleyyt** [comment]
Sorry, I have no time to maintain this repo. So I want to keep it as it is.

**Pull request closed without merging**